sorc/ncep_post.fd/INITPOST_GFS_NEMS_MPIIO.f

```diff
@@ -4013,12 +4005,6 @@ SUBROUTINE INITPOST_GFS_NEMS_MPIIO(iostatusAER)
 !                          ,recname,reclevtyp,reclev,VarName,VcoordName &
 !                          ,ducmass25)
 !     if(debugprint)print*,'sample ',VarName,' = ',ducmass25(isa,jsa)
-
-        if (me == 0) print *,'after aer files reading,mype=',me
-       call nemsio_close(rfile,iret=status)
-       deallocate(tmp,recname,reclevtyp,reclev)
-      end if ! end of aer file read
-
```

Comment:
Why deleted these lines?

Reply:
because those lines were moved to 3962-3963